docs/getting-started/devenv/using-visual-studio-code.md

```diff
@@ -12,19 +12,387 @@ ms.devlang: dotnet
+changes to make to the `project.json` file.
+
+You need to remove the `buildOptions` node from `project.json` so that the build process
+does not emit the startup code:
+
+```js
+"buildOptions" {
+    "emitEntryPoint": true
+}
+```
+
+Next, you'll need to add three dependency nodes. These are the xUnit test library
+project, the .NET core xUnit Test Runner, and the library you wrote in the steps
+above. Open `project.json` and update the dependencies section to the following:
+
+```js
+"dependencies": {
+    "Microsoft.NETCore.App": {
+        "type": "platform",
+        "version": "1.0.0-rc2-3002702"
+    },
+    "xunit": "2.1.0",
+    "dotnet-test-xunit": "1.0.0-rc2-build10025",
+    "library": {
+        "target": "project",
+        "version": "1.0.0.*"
+    }
+}
+```
+
+The `library` node specifies that this dependency should resolve to a project
+in the current workspace. Without explicitly specifying this, it's possible
+that the test project would build against a NuGet package of the same name.
+
+If you try to build at this point, you'll find that the test library does
+not build. That's because the `xunit` NuGet package was built to work
+against a Portable Class Library profile. You need to update the `project.json`
+file to include the portable profiles in the frameworks section:
+
+```js
+"frameworks": {
+    "netcoreapp1.0": {
+        "imports": [
+            "dnxcore50",
+            "portable-net45+win8"
+        ]
+    }
+}
+```
+
+Finally, you need to add a `testrunner` node to `project.json` that
+tells the .NET SDK that you are using the xUnit test runner. Place this
+node between the `version` and the `dependencies` nodes in your `project.json`
+file.
+
+```js
+"testRunner": "xunit",
+```
+
+Now that the dependencies have been properly configured, let's create
+the tests for your library. Rename `Program.cs` to `LibraryTest.cs` and
+replace its contents with the following code:
+
+```csharp
+using Library;
+using Xunit;
+
+namespace TestApp
+{
+    public class LibraryTests
+    {
+        [Fact]
+        public void TestThing() {
+            Assert.Equal(42, new Thing().Get(19.23));
+        }
+    }
+}
+```
+
+Now, run `dotnet restore`, `dotnet build` and `dotnet test`.
+The xUnit console test runner will run the one test, and report
+that it is passing. 
+
+### Writing the console app
+
+In your terminal, cd to the `golden/src/app` directory. Run `dotnet new`
+to create a new console application.
+
+Your console application depends on the library you built and tested
+in the previous steps. You need to indicate that by editing `project.json`
+to add this dependency.  In the `dependencies` node, add the `Library`
+node below the `Microsoft.NetCore.App` node as follows:
+
+```js
+"dependencies": {
+    "Microsoft.NetCore.App": {
+        "type": "platform",
+        "version": "1.0.0-rc2-3002702"
+    },
+    "library": {
+        "target": "project",
+        "version": "1.0.0-*"
```

Comment:
We should drop the splat in here.  Even though it's fine and only a project-to-project reference, we do want to discourage the use of splat strings.